driver_cpl/bld/build-namelist

```diff
@@ -287,14 +287,11 @@ foreach my $file (@files) {
     my @e = $xml->elements_by_name('entry');
     while ( my $e = shift @e ) {
 	my %a = $e->get_attributes();
+	my $val = $a{value};
```

Comment:
$val is a local variable that does not appear to be used in this while loop - so why is it introduced?